app/controllers/users_controller.rb

```diff
@@ -1,5 +1,11 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
```

Comment:
* I suggest you make the views for this specific controller display as stated in the project requirement you should make your views functions as the wireframe given 
![image](https://user-images.githubusercontent.com/52497246/144685291-b0ff03e6-f5d6-4448-825f-be0e6362c19d.png)